### DISCUSSION TITLE
Discovery of the target repository in Simple Release examples

### DISCUSSION THREAD

**Discussion by scordio** [jreleaser/jreleaser#992] [Q&A]
I am evaluating JReleaser for a use case where dozens of [Maven CI friendly](https://maven.apache.org/maven-ci-friendly.html) repositories share a centralized GitLab pipeline. In this pipeline, the release job performs `mvn deploy` and then creates a GitLab release entry using the GitLab API.

As a start, I'd like to delegate the GitLab release entry creation to JReleaser, especially to take advantage of the changelog generation.

Looking at the [Simple Release](https://jreleaser.org/guide/latest/examples/simple-release.html) examples, I noticed that the page mentions:

> The following configuration creates and tags a release at the chosen Git repository, in this case it will be https://github.com/duke/app.

but I couldn't spot any reference to https://github.com/duke/app in the various configurations.

Is the repository discovered automatically, not relying on any config file or parameter? This would help me understand the impact on such an ecosystem, especially if changes in each repository are required.

Apologies in advance in case this is already explained somewhere else in the docs and I missed it 🙂

**Comment by aalmiray** [comment]
`https://github.com/duke/app` is a dummy value. The docs could make it more explicit.
Perhaps https://github.com/aalmiray/app/ cold be used as a reference.

**Reply by scordio** [reply]
Sure, the fact that it's a dummy value was clear to me but I couldn't see this value in any of the config snippets [here](https://jreleaser.org/guide/latest/examples/simple-release.html), so I was wondering how the target repository is taken.

**Reply by aalmiray** [reply]
Right. The wording of the paragraph before the snippet should be more explicit.
